Src/SwqlStudio/app.manifest

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
```

Comment:
I'm not sure about the implications if the name and/or version don't match here.  The name is a simple fix.  Version might require a little extra work in the build scripts.

Reply:
Good catch. I'm not comfortable with having those values present but wrong. I did some experiments to see what was the minimum change required to get high DPI working, and it turns out that `<TargetFramework>net472</TargetFramework>` is sufficient. So I reduced the PR to just that.

Reply:
I made the manifest change and the app.config change because https://docs.microsoft.com/en-us/dotnet/framework/winforms/high-dpi-support-in-windows-forms said to, but that may only matter if you have multiple monitors at different DPIs.